pkg/build/builder/util.go

```diff
@@ -46,26 +56,83 @@ func readNetClsCGroup(reader io.Reader) string {
 	names := []string{"net_cls", "cpu"}
 	for _, group := range names {
 		if value, ok := cgroups[group]; ok {
-			return value
+			return value, containerType
+		}
+	}
+
+	return "", containerType
+}
+
+// readResolveConfHostPath determines the path to the resolv.conf file for this
+// container, as it exists on the host machine.  (/etc/resolv.conf is mounted
+// into the container from the host path).
+func readResolvConfHostPath() (string, error) {
+	// find the /etc/resolv.conf host path based on what is mounted into this
+	// container.
+	resolvConf, err := os.Open("/proc/1/mountinfo")
```

Comment:
This is going to fail when we turn on shared pid namespace, isn't it?

Reply:
no idea.  will containers no longer have their own /proc filesystem with their own pid 1 at that point?

Reply:
Yeah, this won't work with pid namespace sharing. Cleaner way is using crio inspect endpoint but hopefully, we can move away from docker before we have to do that.